index.bs

```diff
@@ -550,12 +550,15 @@ which would have issues of its own.
 
 <h3 id="concepts-sensors">Sensors</h3>
 
-A [=sensor=] measures different physical quantities
+By <dfn id="concept-device-sensor">device sensor</dfn> this specification implies a device's
```

Comment:
The term device sensor refers to a device's

Reply:
done